src/test/java/com/ibm/watson/modelmesh/example/ExampleModelRuntime.java

```diff
@@ -62,7 +54,7 @@
 /**
  * An example Model Serving runtime for use with the model-mesh framework
  */
-public class ExampleModelRuntime extends ModelServerImplBase {
+public class ExampleModelRuntime extends ModelRuntimeGrpc.ModelRuntimeImplBase { // ModelServerImplBase {
```

Comment:
This is updating the mock runtime used in the tests to use the non-legacy internal management interface (methods are the same, only the name changed).